src/contexts/shoppingListsContext.tsx

```diff
@@ -78,7 +90,8 @@ export const ShoppingListsContext = createContext<ShoppingListsContextType>({
 })
 
 export const ShoppingListsProvider = ({ children }: ProviderProps) => {
-  const { user, token, authLoading, requireLogin } = useGoogleLogin()
+  const { user, token, authLoading, requireLogin, withTokenRefresh } =
```

Comment:
Let's remove the two occurrences of `user` in this file as well since they aren't necessary either.